### PR TITLE
Improve theme system for easier overriding

### PR DESCRIPTION
**Pull request by benogle**
* More theme color vars
* Adds ability to override syntax theme
* More docs around theming

### CODE REVIEW

File: src/themes/default/stylesheets/main.scss [hunk 1]
```diff
@@ -67,7 +68,7 @@ $background-arguments: #fafbfc;
 
 @import 'custom';
 @import 'base';
-@import 'monokai';
+@import 'syntax-highlighting';
```
Comment (benogle) [inline src/themes/default/stylesheets/main.scss:71]: This could just be named `syntax`. @newhouse wdyt?

Comment (newhouse) [reply src/themes/default/stylesheets/main.scss:71]: Good by me!

**Review by newhouse — APPROVED** [review]
This looks great to me...with my only concern being around what will change for existing users when this gets merged. I think the answer is "nothing", from the look of it? It just adds more ability for easy variable overrides in `custom.scss`, but does not move or change things from what they were, right?

**Comment by benogle** [comment]
> with my only concern being around what will change for existing users when this gets merged. I think the answer is "nothing", from the look of it? 

Yeah, there will be no changes to the themes. I just used a few variables in more places so they could be overridden by custom.scss

**Pull request merged**